main.tf

```diff
@@ -68,5 +68,5 @@ module "metrics" {
 
 module "remote-test" {
     source            = "git::https://github.com/yashg-ti/Test.git"
```

Comment:
remind me. What path is the yash-ti/Test.git content extracted to? Is it in the workspace? or is it in the .cloudfix-linter dir?

Reply:
workspace, .terraform dir. Download is a result of terraform init, we don't download anything